packages/core/src/components/BaseDropdown/BaseDropdown.styles.tsx

```diff
@@ -183,8 +183,12 @@ export const StyledExtension = styled(
     $backgroundColor: string;
```

Comment:
We need to remove `$backgroundColor` from the types definition and from the `StyledExtension` components used inside the `BaseDropdown.tsx` file.